File: backend/src/main/java/ru/home/shop/service/candy/CreateCandyCommand.java
package ru.home.shop.service.candy;

import lombok.Builder;
import lombok.Value;

import java.math.BigDecimal;
import java.util.UUID;

@Builder
@Value
public class CreateCandyCommand {
    UUID id;
    String name;
    String firm;
    BigDecimal price;
    Double order;
}

File: backend/src/main/kotlin/io/github/artemy/osipov/shop/service/candy/CreateCandyCommand.kt
package io.github.artemy.osipov.shop.service.candy

import java.util.UUID
import java.math.BigDecimal

data class CreateCandyCommand(
    val id: UUID,
    val name: String,
    val firm: String,
    val price: BigDecimal,
    val grams: BigDecimal?,
    val order: BigDecimal
)